### ISSUE TITLE
Screen reader issues - round 1

### ISSUE THREAD

**Issue by Jay-Neighbours-I72** [truetandem/e-QIP-prototype#2564]
Using Chromevox in Chrome and Voiceover in Safari:

- [x] Headlines, body text, and labels are not being read. When tabbing to an input screen reader only says "Edit text"
Maybe labels, etc. are missing? Ex: https://webaim.org/articles/jaws/forms.htm

- [x] Tabbing not consistent across sections (ex: Other names used skipped full content on first tab when landing on this screen after using next button)

- [x] Skip to main content link not working

- [x] When agreements/instructions overlay boxes are on the screen: screen reader takes user through the full content behind the overlay first, then skips content of overlay and goes to overlay buttons.

**Comment by bmallred** [comment]
@Jay-Neighbours-I72 notice it was never checked off... i don't think I meant for this one to be closed (automagic github stuff). I do have the resolution for this and will make sure to tag it on the referenced issue!
